Scrivito/scrivito_example_app_js
# Prerender in Node.js

Get rid of the express/puppeteer dependency in favor of native Node.js prerendering.

I tried to make the steps as clear as possible, including some temporary intermediate changes.

Apart from regular feedback, we need to discuss when to merge.
The SDK docs aren't ready yet, but they may be helpful to explain the changes we made here.

### Manual test
To compare the outcome:
```bash
git checkout master
SCRIVITO_ORIGIN=http://localhost:8080 npm run prerender
cp -R buildPrerendered tmp/buildPrerenderedMaster
git checkout dave-node-js-prerendering
SCRIVITO_ORIGIN=http://localhost:8080 npm run prerender
opendiff tmp/buildPrerenderedMaster buildPrerendered
```
It should be identical except for some preload dump diffs.
